src/ucm/event/event.c

```diff
@@ -396,21 +519,24 @@ static ucs_status_t ucm_event_install(int events)
     }
 
     /* TODO lock */
-    status = ucm_mmap_install(native_events);
-    if (status != UCS_OK) {
-        ucm_debug("failed to install mmap events");
-        goto out_unlock;
-    }
+    if (native_events) {
```

Comment:
why need this if?

Reply:
removed., not relevant for this PR.  I had this to fix some issue with complete feature PR.  will revisit it in upcoming PRs if this needed,